SP5WWP/m17-packet/m17-packet-encode.c

```diff
@@ -199,6 +223,7 @@ int main(int argc, char* argv[])
         fprintf(stderr, "Packet data too short. Exiting...\n");
         return -1;
     }
+    num_bytes++; //increment by one to include the terminating byte
```

Comment:
Question - you pass `-n 25` for the example above. But according to [this](https://github.com/M17-Project/M17_Implementations/blob/main/SP5WWP/tests/m17-sms.sh) script, I think it should be `-n 26`, without the need to increment anything more?